content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,106 @@
+system, such as emails, leads/opportunities, project tasks, helpdesk tickets, and more.
 
-.. image:: media/generate_from_website06.png
+The free *Form Builder* module can be installed from either the apps page or automatically when
+added to a page in the Website Builder. Adding a whole new form can be great if you are
+soliciting certain information that needs filtering, such as suggestions, bug reports, etc.
+
+.. image:: generate_from_website/form-building-blocks.png
+   :align: center
+   :alt: Form Builder building blocks
+
+However, if editing the available predefined options is all that’s necessary, editing the Contact
+Form itself is the best option.
+
+Editing contact form fields
+---------------------------
+In edit mode, click on the field to change, and you can edit any of the following information for
+the specific field:
+
+- **Type**: Choose a custom field option or an existing field. Examples include phone, file
+  upload, language, etc.
+- **Input Type**: This determines the type of entry customers should input. Available options are
+  text, email, telephone, and URL.
+- **Label Name**: Type in the display name to show users what information is needed from them.
+- **Label Position**: Choose the way the label is aligned with the rest of the form. The label
+  can be hidden, above the field, to the far left of the field, or right adjusted and closer to the
+  field.
+- **Required**: Toggleable option for information that you absolutely need entered
+- **Hidden**: Toggleable option to hide the given field for this form or not
+- **Shown on Mobile**: Toggleable option to show the field to users on mobile devices.
+
+.. image:: generate_from_website/editable-field-options.png
    :align: center
+   :alt: Editable field options
 
-By default, any new contact form will send an email. You can switch to
-lead/opportunity generation in *Change Form Parameters*.
+By default, any new contact form will send an email. You can change this to lead/opportunity
+generation in edit mode by clicking any of the form’s fields, and under the form options,
+selecting **Create an Opportunity** as the action.
 
-.. note::
-   If the same visitor uses the contact form twice, the second
-   entry will be added to the first lead/opportunity in the chatter.
+.. tip::
+    If leads are activated in your CRM settings, selecting **Create an Opportunity** will generate
+    a lead.
 
 Generate leads instead of opportunities
 =======================================
 
 When using a contact form, you should use a qualification step
-before assigning the form to the right sales people. To do so, activate *Leads*
-in CRM settings and refer to :doc:`convert`.
-
-.. |image0| image:: ./media/generate_from_website01.png
-   :width: 1.04401in
-   :height: 1.16146in
-.. |image1| image:: ./media/generate_from_website02.png
-   :width: 1.43229in
-   :height: 1.16244in
+by activating *Leads* in CRM settings and refer to :doc:`../sales/crm/acquire_leads/convert`.
```

Comment:
The correct `:doc:` directive link is `` :doc:`convert` ``. The way relative links work is explained under my 'main' comment.